pkgs/misc/vim-plugins/default.nix

```diff
@@ -439,6 +439,19 @@ rec {
     };
   };
 
+  vim-jinja = buildVimPlugin {
+    name = "vim-jinja";
```

Comment:
please add version string suffix, if there isn't a git tag we use something like, `-2015-04-23`